pkg/segment/query/iqr/intermediateQueryResult_test.go

```diff
@@ -642,7 +642,7 @@ func Test_getFinalStatsResults(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, bucketCount, actualBucketCount)
 	assert.Equal(t, groupByCols, actualGroupByCols)
-	assert.Equal(t, measureFuncs, actualMeasureFuncs)
+	assert.ElementsMatch(t, measureFuncs, actualMeasureFuncs)
```

Comment:
Why don't we necessarily expect them in order?

Same comment below

Reply:
We are looping over the [knownValues](https://github.com/siglens/siglens/pull/1792/files#diff-24e928c23b70a5764f8ad7cb3869c8a7f7114e941ca26c895dd7d94dc9af025cR967) map to gather the measureCols since it could include created columns from other commands like eval, bin, etc. Map iteration order is not deterministic so used the ElementsMatch instead of Equal.

Reply:
`measureCols` can contain things that aren't measure columns from a stats? That seems odd. What's an example of when that would happen?

Reply:
`* | stats count as cnt | eval n = 1`, here the `n = 1` has to be returned as a measureCol. So, for segmentStatsCmd and groupByCmd, we have a fixed way of sending data that is in form of measureValues, for a query like this if there is a create column like n in this case, it has to be sent as a measureCol.